src/tiled.lisp

```diff
@@ -587,8 +587,8 @@
               (or (%find-tile (mask-field (byte 29 0) (tobject-gid tobject)) tilesets)
                   (break)))))))
 
-(defun %load-objects (tobjects)
-  (mapcar #'%load-object tobjects))
+(defun %load-objects (tobjects resource-loader)
+  (mapcar #'(lambda (object) (%load-object object resource-loader)) tobjects))
```

Comment:
Please remove the `#'` from the `lambda`

Reply:
Sorry, I don't know where I get this weird `#'(lambda)` reflex from 😅

Reply:
No worries. It doesn't hurt anything, but the less there is of it in the wild, the less chances that new people will adopt it, so I rather squash them when I can 😄